### DISCUSSION TITLE
✨ [Feature]: 通过iframe将tinyEngine嵌入到项目中，项目配置组件的json用postMessage传给tinyEngine，如何正确渲染至画布

### DISCUSSION THREAD

**Discussion by leeron-obj** [opentiny/tiny-engine#430] [Q&A]
### What problem does this feature solve

通过iframe将tinyEngine嵌入到项目中，项目配置组件的json用postMessage传给tinyEngine，如何正确渲染至画布

### What does the proposed API look like

 无

### What is your project name

无

[a comment by an automated account is not shown]

**Comment by chilingling** [comment]
not issue, transfer to discussion.
@leeron-obj

**Comment by chilingling** [comment]
当前TinyEngine不支持从外部接收 json。如果需要改造 TinyEngine 从外部接收 json。建议看看 `package/controller/src/useResource.js` 文件的相关逻辑进行改造。
1. 从外部接收物料信息
2. 与当前物料信息合并
3. 通知依赖进行更新

@leeron-obj
